packages/react/src/composite/root/useCompositeRoot.ts

```diff
@@ -368,10 +368,10 @@ export function useCompositeRoot(params: UseCompositeRootParameters) {
 
 function isModifierKeySet(event: React.KeyboardEvent<any>, ignoredModifierKeys: ModifierKey[]) {
```

Comment:
It should work without `any` here as well.

```suggestion
function isModifierKeySet(event: React.KeyboardEvent, ignoredModifierKeys: ModifierKey[]) {
```